pom.xml

```diff
@@ -17,14 +17,23 @@
         <version>2.2.4.RELEASE</version>
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
+    <repositories>
+        <!-- To create Swagger API docs with spark-java -->
+        <repository>
+            <id>manusant-beerRepo</id>
+            <url>https://packagecloud.io/manusant/beerRepo/maven2</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
     <build>
         <resources>
             <resource>
                 <directory>src/main/resources</directory>
-                <filtering>true</filtering>
-                <includes>
-                    <include>logback.xml</include>
-                </includes>
```

Comment:
Does this mean the logback file isn't used in the shaded jar? It seems like we would want this to persist, but I'm not really sure if this does anything...

Reply:
If you set it that way, then only the files in the `includes` list are included.  Now the build includes all the files under resources.